components/script/dom/identityhub.rs

```diff
@@ -2,46 +2,50 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-use webgpu::wgpu::{AdapterId, Backend, DeviceId, IdentityManager, SurfaceId};
+use webgpu::wgpu::{hub::IdentityManager, id::AdapterId, Backend};
 
 #[derive(Debug)]
 pub struct IdentityHub {
-    adapters: IdentityManager<AdapterId>,
-    devices: IdentityManager<DeviceId>,
+    adapters: IdentityManager,
+    devices: IdentityManager,
 }
 
 impl IdentityHub {
-    fn new(backend: Backend) -> Self {
+    fn new() -> Self {
         IdentityHub {
-            adapters: IdentityManager::new(backend),
-            devices: IdentityManager::new(backend),
+            adapters: IdentityManager::default(),
+            devices: IdentityManager::default(),
         }
     }
 }
 
 #[derive(Debug)]
 pub struct Identities {
-    surface: IdentityManager<SurfaceId>,
+    surface: IdentityManager,
     hub: IdentityHub,
+    backends: [Backend; 3],
```

Comment:
There is a problem here when multiple backends are involved: we can't use the same `IdentityManager` of, say, devices, for multiple backends. Each backend should have its own set of identity managers, except for `surface` things.

Reply:
Thanks for pointing that out @kvark. I've changed `Identities` to have a hub for each backend in the fixup commit.